app/presenters/formats/popular_links_presenter.rb

```diff
@@ -0,0 +1,47 @@
+module Formats
+  class PopularLinksPresenter
+    # Unlike presenter for other models which inherit from editions model,
+    # PopularLinksPresenter do not inherit from EditionFormatPresenter
+    # This is because Edition presenter has dependency on Artifact and
+    # Popular links are not coupled with Artefact unlike any other models.
+    # Hence the difference
+    def initialize(popular_links_edition)
+      @popular_links_edition = popular_links_edition
+    end
+
+    def render_for_publishing_api(*)
+      required_fields.merge optional_fields
+    end
+
+  private
+
+    attr_reader :popular_links_edition
+
+    def required_fields
+      {
+        title: popular_links_edition.title,
+        schema_name: "link_collection",
+        document_type: "link_collection",
+        public_updated_at: public_updated_at.rfc3339(3),
+        publishing_app: "publisher",
+        rendering_app: "frontend",
+        details:,
+      }
+    end
+
+    def details
+      {
+        link_items: popular_links_edition.link_items,
+      }
+    end
+
+    def optional_fields
+      access_limited = { auth_bypass_ids: [popular_links_edition.auth_bypass_id] }
+      { access_limited:, public_updated_at: public_updated_at.rfc3339(3) }
```

Comment:
`public_updated_at` is also in hash returned by `required_fields`—does it make sense to be in both? Is it required, or optional?

Reply:
good catch, In my head, I had moved it to optional fields.